addons/bus/models/bus.py

```diff
@@ -110,20 +110,23 @@ def _sendmany(self, notifications):
             })
         self.sudo().create(values)
```

Comment:
> @seb-odoo We could also imagine buffering the values in a mechanism like `precommit`. This would allow to make `_sendmany` obsolete...

+1 on my side (a bit like tracking values that are accumulated then sent once)

Reply:
It's a good idea, the main concern I had with the PR was that it introduced multiple ways to send that are different, but if we can optimize internally instead it and keep one simple syntax it would be nice!

Reply:
I believe the fact that I am adding a new way to send is mitigated by the fact that
* I deprecated `_sendmany` by batching `_sendone` by default
* The only `_sendone` remaining are in PoS, Web Editor, Survey, and 2 in Mail. All these have a hand crafted channel that deserves special care and security reviews; they are using the non-secure API that we should avoid if possible.

Reply:
I believe those in mail can either be converted or be gone (see other comments).